packages/experiments/src/components/signals/Signals.scss

```diff
@@ -77,6 +77,10 @@
 }
 
 .mention {
+  position: relative;
+
+  top: 0.2em;
```

Comment:
this is very odd css...

Reply:
Well, if someone could just fix the icon font so the glyphs lined up better that would be nice!

Reply:
can you add a comment then about your reasoning here?